pandas/tests/io/formats/test_format.py

```diff
@@ -46,6 +49,13 @@
 use_32bit_repr = is_platform_windows() or not IS64
 
 
+def get_local_am_pm():
```

Comment:
Probably can just inline this since it's used once

Reply:
Indeed for this PR, however I am going to reuse it in the next PR concerning strftime perf  :)
Let me know if still for this PR you prefer inlining.